node_build/make.js

```diff
@@ -144,7 +144,7 @@ Builder.configure({
     // with NEON on the BBB, or want to set -Os (OpenWrt)
     // Allow -O0 so while debugging all variables are present.
     if (CFLAGS) {
-        var cflags = CFLAGS.split(' ');
+        var cflags = CFLAGS.split.trim().split(/\s+/);
```

Comment:
`CFLAGS.split.trim().split(/\s+/);` won't do what you're expecting. I think you want CFLAGS.trim().split(/\s+/);